nemo_curator/utils/distributed_utils.py

```diff
@@ -313,9 +332,13 @@ def read_single_partition(
             read_kwargs["dtype"] = (
                 ast.literal_eval(input_meta) if type(input_meta) == str else input_meta
             )
+            # because pandas doesn't support `prune_columns`, it'll always return all columns even when input_meta is specified
+            # to maintain consisntency we explicitly set `io_columns` here
```

Comment:
```suggestion
            # because pandas doesn't support `prune_columns`, it'll always return all columns even when input_meta is specified
            # to maintain consistency we explicitly set `io_columns` here
```